products/microos/main.pm

```diff
@@ -68,12 +68,21 @@ sub load_feature_tests {
         loadtest 'console/kubeadm';
     }
     elsif (check_var 'SYSTEM_ROLE', 'container-host') {
-        my $args = OpenQA::Test::RunArgs->new();
-        my $runtime = get_required_var('CONTAINER_RUNTIME');
-        $args->{runtime} = $runtime;
+        my $args_podman = OpenQA::Test::RunArgs->new();
+        my $args_docker = OpenQA::Test::RunArgs->new();
+        $args_podman->{runtime} = 'podman';
+        $args_docker->{runtime} = 'docker';
```

Comment:
I don't understand this. Is it even related? Why are you getting rid of `CONTAINER_RUNTIME` variable?

Reply:
For MicroOS, it's a bit special. I wanted to keep it simple and run podman and docker tests in one shot regardless of that variable. We are not calling main_containers.pm in this schedule, that variable is not needed. 
But if it's confusin, I can add it back and we can do `CONTAINER_RUNTIME=podman,docker`

Reply:
I would be for `CONTAINER_RUNTIME=podman,docker`. Can we also have a ticket where we move container-related logic from microos to containers?